allofplos/tests/test_unittests.py

```diff
@@ -110,7 +83,7 @@ def test_class_doi1(self):
         self.assertEqual(article.get_fn_dict(), {'coi001': 'The authors have declared that no competing interests exist.'}, 'get_fn_dict does not transform correctly for {}'.format(article.doi))
         self.assertEqual(article.get_plos_journal(), "PLOS ONE", 'get_plos_journal does not transform correctly for {}'.format(article.doi))
         self.assertEqual(article.get_related_dois(), {}, 'get_related_dois does not transform correctly for {}'.format(article.doi))
-        self.assertEqual(article.get_url(), "http://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0185809&type=manuscript", 'get_url does not transform correctly for {}'.format(article.doi))
+        self.assertEqual(article.get_page(page_type='assetXMLFile'), "http://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0185809&type=manuscript", 'get_page does not transform correctly for {}'.format(article.doi))
```

Comment:
I can’t actually comment below this line but, in general all the longer these assert equals are fairly hard to parse, is there a way to simplify them?

Reply:
L88 is especially bad in this regard.

Reply:
yeah I'm not sure. Line breaks?

Reply:
Line breaks… generalising the error message you want to return and defining it as a format string that you use in all of these cases e.g.,   

```python
template = '{name} does not transform correctly for {doi}'
template.format(doi=article.doi, name='get_id_dict')`
```